Ginger/GingerCore/GeneralLib/General.cs

```diff
@@ -399,6 +400,15 @@ public static bool IsNameAlreadyexists(RepositoryItemBase repositoryItem, string
 
             switch (repositoryItem.GetItemType())
             {
+                case "HTMLReportConfiguration":
+                    if (WorkSpace.Instance.SolutionRepository.GetAllRepositoryItems<HTMLReportConfiguration>().Any(x => x.Name == resultValue))
+                    {
+                        Reporter.ToUser(eUserMsgKey.StaticWarnMessage, "Report Template with same name: " + "'" + resultValue + "'" + " already exists.");
```

Comment:
Use string interpolation to concat string

Reply:
Done